src/SendGrid/SendGridClient.cs

```diff
@@ -76,25 +106,10 @@ public SendGridClient(IWebProxy webProxy, string apiKey, string host = "https://
         /// <param name="host">Base url (e.g. https://api.sendgrid.com)</param>
         /// <param name="requestHeaders">A dictionary of request headers</param>
         /// <param name="version">API version, override AddVersion to customize</param>
-        /// <param name="urlPath">Path to endpoint (e.g. /path/to/endpoint)</param>
         /// <returns>Interface to the SendGrid REST API</returns>
-        public SendGridClient(string apiKey, string host = null, Dictionary<string, string> requestHeaders = null, string version = "v3", string urlPath = null)
+        public SendGridClient(string apiKey, string host = null, Dictionary<string, string> requestHeaders = null, string version = "v3")
```

Comment:
Please re-add the urlPath parameter so we can avoid a breaking change. It allows for the user to specify a default endpoint path.